config/locales/en.yml

```diff
@@ -391,570 +408,624 @@ en:
-            text: "Supporting the mental health and wellbeing of children and young people"
-            href: "https://www.gov.uk/government/publications/covid-19-guidance-on-supporting-children-and-young-peoples-mental-health-and-wellbeing/guidance-for-parents-and-carers-on-supporting-children-and-young-peoples-mental-health-and-wellbeing-during-the-coronavirus-covid-19-outbreak"
-          - id: "0108"
-            text: "Additional information about taking care of your mental health (NI direct)"
-            href: "https://www.nidirect.gov.uk/articles/coronavirus-covid-19-taking-care-your-mental-health-and-wellbeing"
-            show_to_nations:
-              - Northern Ireland
-          - id: "0109"
-            text: "Additional information about taking care of your mental health (Public Health Wales)"
-            href: "https://phw.nhs.wales/topics/latest-information-on-novel-coronavirus-covid-19/staying-well-at-home/how-are-you-feeling/"
-            show_to_nations:
-              - Wales
-          - id: "0110"
-            text: "Additional information about coronavirus and your mental wellbeing from the Scottish Association for Mental Health"
-            href: "https://www.samh.org.uk/about-mental-health/self-help-and-wellbeing/coronavirus-and-your-mental-wellbeing"
-            show_to_nations:
-              - Scotland
+        - id: '0103'
+          text: Where to get urgent help for mental health
+          href: https://www.nhs.uk/using-the-nhs/nhs-services/mental-health-services/dealing-with-a-mental-health-crisis-or-emergency/
+        - id: '0104'
+          text: Get information on coronavirus and your wellbeing (Mind)
+          href: https://www.mind.org.uk/information-support/coronavirus/coronavirus-and-your-wellbeing/#collapse838f8
+        - id: '0105'
+          text: Looking after your mental health during coronavirus
+          href: https://www.gov.uk/government/publications/covid-19-guidance-for-the-public-on-mental-health-and-wellbeing
+        - id: '0106'
+          text: Looking after your mental health (NHS)
+          href: https://www.nhs.uk/oneyou/every-mind-matters
+        - id: '0107'
+          text: Supporting the mental health and wellbeing of children and young people
+          href: https://www.gov.uk/government/publications/covid-19-guidance-on-supporting-children-and-young-peoples-mental-health-and-wellbeing/guidance-for-parents-and-carers-on-supporting-children-and-young-peoples-mental-health-and-wellbeing-during-the-coronavirus-covid-19-outbreak
+        - id: '0108'
+          text: Additional information about taking care of your mental health (NI
+            direct)
+          href: https://www.nidirect.gov.uk/articles/coronavirus-covid-19-taking-care-your-mental-health-and-wellbeing
+          show_to_nations:
+          - Northern Ireland
+        - id: '0109'
+          text: Additional information about taking care of your mental health (Public
+            Health Wales)
```

Comment:
```suggestion
          text: |
            Additional information about taking care of your mental health (Public
            Health Wales)
```

Reply:
See here for why i think we shouldn't change these: https://github.com/alphagov/govuk-coronavirus-find-support/pull/222/commits/b0c5b0aafdd33f6b983cf66d96c9d34c0e9bfcf6#r430281216